Fix gaps in SPLICE implementation for instructor dashboard

sarangjo opened the pull request:
This PR adds SPLICE-relevant code to parts of the instructor dashboard that currently don't have support for SPLICE interactive exercises.

bnmnetp commented:
I would like to merge the dashboard pieces as others will benefit from them.  I can fix up the change to the else clause.  But I'll wait for you to move this out of draft status.

sarangjo commented on components/rsptx/build_tools/core.py:
```diff
@@ -1007,7 +1007,7 @@ def _process_single_question(
         else:
```
Sounds good, I can remove the `else` block here! Will the `qtype` be properly set to `"splice"` with the updated manifest format?

sarangjo commented:
@bnmnetp I've moved the PR out of draft stage, and removed the `else` block as you suggested from the build tools script. Please let me know what else I need to do to merge this, or feel free to take ownership of merging the PR—whatever you prefer!
